<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyServicesReportSetupDialog</class>
 <widget class="QDialog" name="DailyServicesReportSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cmbPayStatus">
     <item>
      <property name="text">
       <string>не выбрано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>оплачено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>отказано</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>64</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPayStatus">
     <property name="text">
      <string>Состояние оплаты</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinanceSource"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFinanceSource">
     <property name="text">
      <string>Источник финансирования</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DailyServicesReportSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DailyServicesReportSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
